### PR TITLE
Update Clip Geometry Sample

### PR DESCRIPTION
**Pull request by prupani-7**
## Description
PR to add a reset button to the existing `Clip Geometry` sample. https://devtopia.esri.com/runtime/kotlin/issues/1625

## What To Review
MainActivity.kt and the activity_main.xml file

## How to Test
Run the sample using the branch.

### CODE REVIEW

**Review by shubham7109 — CHANGES_REQUESTED** [review]
Clip geometry reset button works well. Just a couple of minor code suggestions:

**Comment by prupani-7** [comment]
@shubham7109 Thanks for the comments, based on our call, the `reset function` was updated to just clear the clipped area Graphics overlay and make the coloradoGraphic visible. Also, the graphicsOverlay and envelopesOverlay are now defined as lazy in global scope

File: clip-geometry/src/main/java/com/esri/arcgismaps/sample/clipgeometry/MainActivity.kt [hunk 1]
```diff
@@ -39,6 +39,9 @@ import com.esri.arcgismaps.sample.clipgeometry.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
 
+    private val graphicsOverlay: GraphicsOverlay by lazy { GraphicsOverlay() }
+    private val envelopesOverlay: GraphicsOverlay by lazy { GraphicsOverlay() }
```
Comment (shubham7109) [inline clip-geometry/src/main/java/com/esri/arcgismaps/sample/clipgeometry/MainActivity.kt:43]:
```suggestion
    // graphics overlay along the Colorado state border
    private val coloradoOverlay: GraphicsOverlay by lazy { GraphicsOverlay() }

    // graphics overlay to contain the clipping envelopes
    private val envelopesOverlay: GraphicsOverlay by lazy { GraphicsOverlay() }
```

**Review by shubham7109 — APPROVED** [review]
Just a minor comment that I think got lost in the reformatting.  The reset button looks nice, and the sample code is much better! 👍

File: clip-geometry/src/main/java/com/esri/arcgismaps/sample/clipgeometry/MainActivity.kt (hunk 1 above)
Comment (prupani-7) [reply clip-geometry/src/main/java/com/esri/arcgismaps/sample/clipgeometry/MainActivity.kt:43]: Yes makes sense to rename to coloradoOverlay for better readability.

**Review by fpearson1111 — APPROVED** [review]
Nice, works well on my device :+1:

**Comment by prupani-7** [comment]
Thanks for the review @fpearson1111 and @shubham7109 . Merging into v.next

**Pull request merged**